app/src/main/java/com/cmput301f20t21/bookfriends/ui/request/RequestActivity.java

```diff
@@ -144,11 +175,96 @@ public void openDialog(int position) {
                 .setPositiveButton(R.string.edit_confirm, new DialogInterface.OnClickListener() {
                     public void onClick(DialogInterface dialog, int id) {
                         Toast.makeText(RequestActivity.this, "Accepted Request", Toast.LENGTH_SHORT).show();
+                        openMapDialog();
```

Comment:
This opens the map even if the accept request fail. We need a success callback to open the map.

Reply:
I think we first should check if user clicks confirm on the map fragment then accept request. If user still cancels on the map fragment, that doesn't mean accept.